draft-ietf-ohai-ohttp.md

```diff
@@ -279,54 +279,78 @@ map display).
 
 {::boilerplate bcp14-tagged}
 
+*[Client]: #dfn-client
+*[Clients]: #dfn-client
+*[Encapsulated Request]: #dfn-enc-req
+*[Encapsulated Response]: #dfn-enc-res
+*[Oblivious Relay Resource]: #dfn-relay
+*[Oblivious Gateway Resource]: #dfn-gateway
+*[Oblivious Relay Resources]: #dfn-relay
+*[Oblivious Gateway Resources]: #dfn-gateway
+*[Target Resource]: #dfn-target
+*[Relay Request]: #dfn-relay-req
+*[Relay Response]: #dfn-relay-res
+*[Gateway Request]: #dfn-gateway-req
+*[Gateway Response]: #dfn-gateway-res
+
 Client:
 
-: This document uses its own definition of client.  When referring to the HTTP
+: This document uses its own definition of Client.  When referring to the HTTP
   definition of client ({{Section 3.3 of HTTP}}), the term "HTTP client" is
   used; see {{http-usage}}.
+  {: anchor="dfn-client"}
 
 Encapsulated Request:
 
 : An HTTP request that is encapsulated in an HPKE-encrypted message; see
   {{request}}.
+  {: anchor="dfn-enc-req"}
 
 Encapsulated Response:
 
 : An HTTP response that is encapsulated in an HPKE-encrypted message; see
   {{response}}.
+  {: anchor="dfn-enc-res"}
 
 Oblivious Relay Resource:
 
 : An intermediary that forwards encapsulated requests and responses between
-  clients and a single Oblivious Gateway Resource.
+  Clients and a single Oblivious Gateway Resource.
+  {: anchor="dfn-relay"}
 
 Oblivious Gateway Resource:
 
 : A resource that can receive an encapsulated request, extract the contents of
   that request, forward it to a Target Resource, receive a response,
   encapsulate that response, then return that response.
+  {: anchor="dfn-gateway"}
 
 Target Resource:
 
 : The resource that is the target of an encapsulated request.  This resource
   logically handles only regular HTTP requests and responses and so might be
   ignorant of the use of Oblivious HTTP to reach it.
+  {: anchor="dfn-target"}
 
 Relay Request:
 
 : An HTTP request from Client to Relay that contains an encapsulated request as the content.
+  {: anchor="dfn-relay-req"}
 
 Relay Response:
 
 : An HTTP response from Relay to Client that contains an encapsulated response as the content.
+  {: anchor="dfn-relay-res"}
 
 Gateway Request:
 
 : An HTTP request from Relay to Gateway that contains an encapsulated request as the content.
+  {: anchor="dfn-gateway-req"}
 
 Gateway Response:
 
 : An HTTP response from Gateway to Relay that contains an encapsulated response as the content.
+  {: anchor="dfn-gateway-res"}
```

Comment:
@chris-wood How would you feel about deleting these Xxx Request/Response definitions.  The index shows that we don't use them at all.

Reply:
If we don't use them then we can drop them. 👍